sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/download/http/HttpConnectionReleaseDownloader.java

```diff
@@ -128,31 +163,32 @@ public void delete() {
     private void showProgressNotification(long currentSize, long totalSize) {
         Notification.Builder builder = getNotificationBuilder();
 
-        // TODO Use the different string.
+        /* TODO: We should not reuse "mandatory" string here. */
         builder.setContentTitle(mContext.getString(R.string.appcenter_distribute_downloading_mandatory_update))
                 .setSmallIcon(mContext.getApplicationInfo().icon)
-                .setPriority(Notification.PRIORITY_LOW)
-                .setProgress((int) (totalSize / 1024), (int) (currentSize / 1024), totalSize <= 0);
+                .setProgress((int) (totalSize / KIBIBYTE_IN_BYTES), (int) (currentSize / KIBIBYTE_IN_BYTES), totalSize <= 0);
```

Comment:
another place with those long to int conversion

Reply:
This is requirement here. The system API wants ints